client/src/components/songDetails/songDetails.css

```diff
@@ -0,0 +1,45 @@
+
+  @import url('https://fonts.googleapis.com/css2?family=ABeeZee:ital@0;1&display=swap');
```

Comment:
better to download the font and put in assets, expecially in mobile first apps (where internet connection can be slow)